tests/contrib/pylons/tests.py

```diff
@@ -51,3 +52,4 @@ def test_init():
     assert middleware.client.config.service_name == "p" * 32
     assert middleware.client.config.secret_token == "a" * 32
     assert middleware.client.config.metrics_interval.total_seconds() == 0
+    elasticapm.get_client().close()
```

Comment:
If we want to make sure that the client gets closed even when the test fails, we'll have to wrap the test in a `try/finally`, and put the `close()` call into the `finally` block.